Source/WebKit/UIProcess/mac/WebViewImpl.mm

```diff
@@ -208,6 +208,22 @@ - (void)setCanShowMediaSelectionButton:(BOOL)canShowMediaSelectionButton;
 @end
 #endif
 
+#if ENABLE(CONTENT_INSET_BACKGROUND_FILL)
+
+@interface WKContentInsetOverlayView : NSView
+@end
```

Comment:
...it also just occurred to me that this is no longer used. I'll just remove it while I'm upstreaming this.